### PR TITLE
ignore any error that has stop_youtube

### LINKED ISSUE #8915: Error transformer: remove some common errors thrown by extensions and ad blockers
These errors have no value, but only spamming our error monitoring.

Some examples:
- `null is not an object (evaluating 'elt.parentNode')`
- `window.stop_youtube is not a function`

### PR DESCRIPTION
**Pull request by erwinmombay**
Closes #8915.

### CODE REVIEW

**Comment by erwinmombay** [comment]
also ran gofmt to fix formatting inconsistencies

File: tools/errortracker/errortracker.go [hunk 1]
```diff
@@ -194,6 +194,14 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if strings.Contains(event.Message, "stop_youtube") ||
+		strings.Contains(event.Exception, "stop_youtube") {
+		w.Header().Set("Content-Type", "text/plain; charset=utf-8")
+		w.WriteHeader(http.StatusOK)
+		fmt.Fprintln(w, "IGNORE\n")
```
Comment (dvoytenko) [inline tools/errortracker/errortracker.go]: Should not we still return No_CONTENT?

Comment (erwinmombay) [inline tools/errortracker/errortracker.go]: that is probably better

**Comment by erwinmombay** [comment]
@dvoytenko can you review again?

**Comment by jridgewell** [comment]
Version bump?

**Comment by erwinmombay** [comment]
@jridgewell done

**Pull request merged**